src/analysis/special_functions/exp_log.lean

```diff
@@ -389,107 +421,149 @@ end
 lemma log_nonpos (hx : 0 ≤ x) (h'x : x ≤ 1) : log x ≤ 0 :=
 (log_nonpos_iff' hx).2 h'x
 
-section prove_log_is_continuous
+lemma strict_mono_incr_on_log : strict_mono_incr_on log (set.Ioi 0) :=
+λ x hx y hy hxy, log_lt_log hx hxy
+
+lemma strict_mono_decr_on_log : strict_mono_decr_on log (set.Iio 0) :=
+begin
+  rintros x (hx : x < 0) y (hy : y < 0) hxy,
+  rw [← log_abs y, ← log_abs x],
+  refine log_lt_log (abs_pos.2 hy.ne) _,
+  rwa [abs_of_neg hy, abs_of_neg hx, neg_lt_neg_iff]
+end
 
-lemma tendsto_log_one_zero : tendsto log (𝓝 1) (𝓝 0) :=
+/-- The real logarithm function tends to `+∞` at `+∞`. -/
+lemma tendsto_log_at_top : tendsto log at_top at_top :=
```

Comment:
Can you make this argument more general (and, perhaps when your refactor of inverse trig functions goes through, apply it to `arctan`/`arctan`/`arcsin` too)?  One way would be to prove that the `filter.map`/`filter.comap` of `at_top` by an `order_iso` is `at_top`, together with (for the trig ones) `Ioo_at_top_eq_nhds_within` or similar.  Same with the next lemma.

Reply:
Note that `log` is an `ite`, not an `order_iso`. This makes applying general facts about `order_iso` much less convenient. I can use `filter.map_at_top_eq_of_gc` as a general fact here.

Reply:
UPD: no, `map_at_top_eq_of_gc` assumes `monotone f`, not `∃ a, ∀ x y ≥ a, x ≤ y → f x ≤ f y`.

Reply:
I added more lemmas about `exp`, and used one of them here.